<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Autenticacion</class>
 <widget class="QDialog" name="Autenticacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>358</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>358</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>358</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tu visa</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../tuvisa.png</normaloff>../tuvisa.png</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="LabelActivacion">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>31</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Cantarell Extra Bold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Registrar aplicacion</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputActivation">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 5px 5px 5px 20px;
color: rgb(0, 85, 127);
border: 1px solid #000080; 
border-radius: 8px;</string>
   </property>
   <property name="placeholderText">
    <string>Ingresa tu TOKEN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonActivation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
background-color: rgb(0, 0, 128);
border-radius: 8px;
color:white;</string>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QLabel" name="AlertActivation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Serif Tamil ExtraBold</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TuVisaLogo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../tuvisa.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
